tests/pact/base64.repl

```diff
@@ -22,30 +24,42 @@
 
 (expect-failure
   "base64 decoding fails on non base64-encoded input"
-  "Could not decode string"
+  "invalid padding near offset 16"
```

Comment:
I'm confused, this is in the DisablePact49 section, which means non-forking ... and the errors are changing? That's a fork of it's own.

Reply:
Good catch!

This is definitely a difference between the old and new codepaths that would have been a forking change, I suppose it passed replay because this particular error hasn't yet happened on chain.

So, I have updated the error-reformatting code so that the old version of this test passes.

The test case will still appear in the diff because I've made it more specific. (`expect-failure` will match when the argument is a substring of the error message. "Could not decode string" is too lenient of a test. I've changed it to "Could not decode string: THE_REASON". I've copied these tests over and run them against `master`, and the more-strict versions pass.